locale/pt-br/docs/guides/dont-block-the-event-loop.md

```diff
@@ -0,0 +1,488 @@
+---
+title: Não bloqueie o Event Loop (ou a Worker Pool)
+layout: docs.hbs
+---
+
+# Não bloqueie o Event Loop (ou a Worker Pool)
+
+## Você deve ler esse guia?
+Se você está escrevendo algo mais complicado que um breve script de linha de comando, ler este guia ajudará você a escrever aplicativos de maior desempenho e mais seguros.
+
+Este documento foi escrito com servidores Node em mente, mas os conceitos são aplicados para aplicações Node complexas também.
+Onde detalhes específicos do sistema operacional variam, este documento é centrado no Linux.
```

Comment:
Do you have the original text please? This is a bit odd

Reply:
https://github.com/nodejs/nodejs.org/pull/2685/commits/52ce8989c1308ba192f86a60e8b959f4b9d0aa92

Reply:
you can be guided by commits